rest/results-tabulation-api/ext/ExtendedElection/ExtendedElectionParliamentaryElection2020/__init__.py

```diff
@@ -919,7 +948,18 @@ def _get_country_entry(row):
             area_key = area_name
 
             def _create_country_tally_sheets(area):
-                return {}
+                all_island_results_tally_sheet_list = [TallySheet.create(
+                    template=tally_sheet_template_all_island_result, electionId=root_election.electionId,
+                    areaId=area.areaId,
+                    metaId=Meta.create({
+                        "areaId": area.areaId,
+                        "electionId": root_election.electionId
+                    }).metaId,
+                    workflowInstanceId=workflow_report.get_new_instance().workflowInstanceId
```

Comment:
Add the workflow_release_report here. Otherwise it won't get the the release option. (Until I get the confirmation on release allowed tallysheets, let's assume this is also in)

Reply:
This is fixed.